osu.Game.Tests/Visual/UserInterface/TestSceneModSelectOverlay.cs

```diff
@@ -457,10 +457,12 @@ private class TestUnimplementedMod : Mod
             public override ModType Type => ModType.Conversion;
         }
 
-        private class TestUnimplementedModOsuRuleset : OsuRuleset
+        private class TestUnimplementedModOsuRuleset : OsuRuleset, ILegacyRuleset
         {
             public override string ShortName => "unimplemented";
 
+            int ILegacyRuleset.LegacyID => -1;
+
```

Comment:
Tests seem to pass without this being present – what's the reason for adding this here?

Reply:
Right, that's no longer required with 92e22c5.